src/database/migrations/2024_02_04_083757_create_place_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('place', function (Blueprint $table) {
+            $table->uuid("id")->primary();
+            $table->string('name');
```

Comment:
uniqueness ? if so `->unique()`
required ? if so `->required()`

Reply:
`->required()` does not exist ?

Reply:
mmmh finally it might not exist.
I assume that each field is considered mandatory unless the `->nullable()` modifier is applied.